packages/block-editor/README.md

```diff
@@ -880,6 +880,18 @@ _Returns_
 
 -   `any`: Returns the value defined for the setting.
 
+### useShouldContextualToolbarShow
```

Comment:
Question: Do we want to publicize this API just now? As soon as we make it public we have to make it backward-compatible. Maybe we can keep it internal for now?

Reply:
I agree with @kevin940726 - this looks useful only for internal use for now.

Reply:
oh I saw later why this is exported, so we can use it in the post editor later. Do we really need this `useShouldContextualToolbarShow` refactoring?

Reply:
At the time I left this review, it was only used in the same package. If it's also used in other packages and it's useful then I don't mind publicizing it 😅 .

Reply:
I guess there's a fine line between whether it should be implemented as a `@wordpress/data` selector or a hook. I don't mind either but would rather keep it consistent. It's just one small thing to consider before we publicizing it :).

Reply:
Yeah, the `useViewportMatch` is the problem with making it a selector. We could, in the future, do the same thing as `useViewportMatch` and update the store when the viewport changes. So it's a good idea to make it public.